lib/maintenance_tasks.rb

```diff
@@ -9,6 +9,11 @@
 require 'pagy'
 require 'pagy/extras/bulma'
 
+require 'maintenance_tasks/progress'
+require 'maintenance_tasks/runner'
+require 'maintenance_tasks/task_data'
+require 'maintenance_tasks/ticker'
```

Comment:
I'm guessing these were in `app/models` to avoid having to explicitly `require` them?